push-build.sh

```diff
@@ -185,8 +186,20 @@ if ((FLAGS_federation)); then
   common::stepheader PUSH FEDERATION
   ############################################################################
   logecho -n "Push federation images: "
-  # FEDERATION_PUSH_REPO_BASE should be set by the calling job (yaml)
-  logrun -s ${KUBE_ROOT}/federation/develop/push-federation-images.sh
+  GCLOUD_PROJECT=$($GCLOUD config list project 2>/dev/null |\
+                   awk '{project = $3} END {print project}')
```

Comment:
It is simpler to use `gcloud config list project --format 'value(core.project)'`